cmd/booster-http/http_test.go

```diff
@@ -27,10 +27,19 @@ func TestNewHttpServer(t *testing.T) {
 	require.NoError(t, err)
 	waitServerUp(t, 7777)
 
-	// Check that server is responding with 200 status code
-	resp, err := http.Get("http://localhost:7777/")
+	// Create a request with Cors header
```

Comment:
The new test looks good - I'd say let's also keep the old test. So first check a regular request returns 200, then check that the CORS request returns the expected headers.